src/client/src/app/site/[site]/availability/edit-services/confirmed/edit-services-confirmed.tsx

```diff
@@ -38,17 +53,99 @@ const EditServicesConfirmed = ({
           ],
         ]}
       />
-      <InsetText>
-        <p>
-          Some booked appointments may be affected by this change. If so, you'll
-          need to cancel these appointments manually.
-        </p>
-        <Link
-          href={`/site/${site.id}/view-availability/daily-appointments?date=${date}&page=1&tab=2`}
-        >
-          Cancel appointments
-        </Link>
-      </InsetText>
+      {changeSessionUpliftedJourneyEnabled === false ? (
+        <InsetText>
+          <p>
+            Some booked appointments may be affected by this change. If so,
+            you'll need to cancel these appointments manually.
+          </p>
+          <Link
+            href={`/site/${site.id}/view-availability/daily-appointments?date=${date}&page=1&tab=2`}
+          >
+            Cancel appointments
+          </Link>
+        </InsetText>
+      ) : hasBookings === false ? (
+        <>
+          <div className="margin-top-bottom">
+            {servicesCount > 1 ? 'These services have' : 'The service has'} been
+            removed.
+          </div>
+          <Link href={`/site/${site.id}/view-availability/week?date=${date}`}>
+            View all bookings for this week
+          </Link>
+        </>
+      ) : chosenAction === 'cancel-appointments' ? (
+        <>
+          <div className="margin-top-bottom">
```

Comment:
I know you didn't introduce this pattern, but I'm not a fan of this for 3 reasons:

Firstly, we should be deviating from the NHS' Frontend framework as little as possible. Something like this should be perfectly achievable within their grid system / container classes etc. 
Secondly, it's not responsive. It hardcodes margins to 16 pixels. 
Thirdly, it's very unclearly named. 

I appreciate this pattern was created in a different change so if you want keep this PR as-is that's okay, I'll fix it up later. If you have the time though I'd love to replace this with something that's A) responsive and B) in-framework